src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png

```diff
@@ -0,0 +1 @@
+../../../../img/chatzilla22.png
```

Comment:
On 02/03/2015 17:17, foudfou wrote:
>
> In src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png 
> <https://github.com/foudfou/FireTray/pull/167#discussion_r25652633>:
>
> > @@ -0,0 +1 @@
> > +../../../../img/chatzilla22.png
>
> why not a single link: chrome/skin/icons/freebsd -> 
> chrome/skin/icons/linux ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/foudfou/FireTray/pull/167/files#r25652633>.
>
Don't understand?
Link each file? That might work.